packages/template-retail-react-app/app/pages/store-locator/index.jsx

```diff
@@ -70,7 +82,35 @@ const StoreLocator = () => {
                     submitForm={submitForm}
                     storesInfo={storesInfo}
                     searchStoresParams={searchStoresParams}
+                    setSearchStoresParams={setSearchStoresParams}
+                    setUserHasSetManualGeolocation={setUserHasSetManualGeolocation}
                 />
+                {searchStoresParams.limit < numStores && searchStoresParams.limit < 200 ? (
+                    <Box marginTop="10px">
+                        <Button
+                            key="load-more-button"
+                            onClick={() => {
+                                setSearchStoresParams({
+                                    ...searchStoresParams,
+                                    limit:
+                                        searchStoresParams.limit + 10 <= 200
```

Comment:
it's unclear what's the meaning of `200` and `10`, it would be nice to create named variable for these values so it's easier to understand